### PR TITLE
[5.5] Workaround the actor runtimes unwillingness to deal with priorities properly

### PR DESCRIPTION
**Pull request by aschwaighofer**
rdar://79378627

### CODE REVIEW

**Comment by aschwaighofer** [comment]
main PR: https://github.com/apple/swift/pull/37939

**Comment by aschwaighofer** [comment]
@swift-ci Please test

File: test/Concurrency/Runtime/actor_detach.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// RUN: %target-run-simple-swift(-Xfrontend -enable-experimental-concurrency) | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
```
Comment (DougGregor) [inline test/Concurrency/Runtime/actor_detach.swift:4]: The Linux test failure implies that we need a 
```
// REQUIRES: foundation
```
on this

**Comment by aschwaighofer** [comment]
@swift-ci Please test

**Review by DougGregor — APPROVED** [review]

**Review by airspeedswift — APPROVED** [review]

**Pull request merged**